lib/Mojolicious/Static.pm

```diff
@@ -201,6 +213,23 @@ Directories to serve static files from, first one has the highest precedence.
   # Add another "public" directory with higher precedence
   unshift @{$static->paths}, '/home/sri/themes/blue/public';
 
+=head2 special
+
+At startup, Mojolicious enumerates its internal bundled, static files and keeps
+them in this hash reference; for example, the key C<favicon.ico> will have as
+its value the full pathname to Mojolicious's default favicon.  An application
+can examine or change these values. For example, a Mojolicious::Lite app could:
+
+  delete app->static->special->{'favicon.ico'};
+
+and then proceed to serve favicon.ico from a route. Or:
+
+  app->static->special->{'mojo/jquery/jquery.js'} =
+    app->home->child('resources', 'jquery.js');
+
+would override the built-in jQuery for error pages with one in the app's
+C<resources> directory.
```

Comment:
This is not the correct documentation format for attributes. Please replicate the style of the surrounding documentation.